identity/client/e2e/tests/roles.spec.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { expect } from "@playwright/test";
+import { test } from "../text-fixtures";
+import { Paths } from "../utils/paths";
+import { relativizePath } from "../utils/relativizePaths";
+import { LOGIN_CREDENTIALS } from "../utils/constants";
+import { waitForItemInList } from "../utils/waitForItemInList";
+
+const NEW_Role = {
+  id: "testRoleID",
+  name: "Test role",
+};
+
+test.beforeEach(async ({ page, loginPage }) => {
+  await page.goto(relativizePath(Paths.roles()));
+  await loginPage.login(LOGIN_CREDENTIALS);
+  await expect(page).toHaveURL(relativizePath(Paths.roles()));
+});
+
+// TODO: reenable when https://github.com/camunda/camunda/issues/27220 is resolved
+test.describe.skip("roles CRUD", () => {
+  test("creates a role", async ({ page, rolesPage }) => {
+    await expect(
+      rolesPage.rolesList.getByRole("cell", { name: "Admin" }),
+    ).toBeVisible();
+
+    await rolesPage.createRoleButton.click();
+    await expect(rolesPage.createRoleModal).toBeVisible();
+    await rolesPage.createIdField.fill(NEW_Role.id);
+    await rolesPage.createNameField.fill(NEW_Role.name);
+    await rolesPage.createRoleModalCreateButton.click();
+    await expect(rolesPage.createRoleModal).not.toBeVisible();
+
+    const item = rolesPage.rolesList.getByRole("cell", {
+      name: NEW_Role.name,
+    });
+
+    await waitForItemInList(page, item);
+  });
+
+  test("deletes a role", async ({ page, rolesPage }) => {
+    await expect(
+      rolesPage.rolesList.getByRole("cell", { name: NEW_Role.name }),
+    ).toBeVisible();
+
+    await rolesPage.deleteRoleButton(NEW_Role.name).click();
+    await expect(rolesPage.deleteRoleModal).toBeVisible();
+    await rolesPage.deleteRoleModalDeleteButton.click();
+    await expect(rolesPage.deleteRoleModal).not.toBeVisible();
+
+    await expect
+      .poll(
+        async () => {
+          await page.reload();
+          await page.waitForTimeout(1000); // this timeout is needed to give time for the table to load when the page loads, regular assertions do not have the same effect
+          const isVisible = await rolesPage.rolesList
+            .getByRole("cell", { name: NEW_Role.name })
+            .isVisible();
+          return isVisible;
+        },
+        {
+          timeout: 10000,
+        },
+      )
+      .toBeFalsy();
```

Comment:
This looks like a duplicate from what's already being done in the `waitForItemInList` below, I think we can remove it. It might have been caused because I forgot one of these in the Users test originally, I've since removed it in the Groups tests PR.